### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by duylong192**

### CODE REVIEW

File: app/Block/cart.twig [hunk 1]
```diff
@@ -8,58 +8,77 @@ http://www.lockon.co.jp/
 For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 #}
-{% set Carts = get_all_carts() %}
-{% set totalPrice = get_carts_total_price() %}
+<script> 
+$(function() {
+    $('.btn-submit.red').on('click', function() {
```
Comment (tamnguyentps) [inline app/Block/cart.twig:14]: Missing "off('click')"

File: app/Block/cart.twig [hunk 2]
```diff
@@ -8,58 +8,77 @@ http://www.lockon.co.jp/
...
+        $('.btn-submit.grey').removeClass('d-none');
+        $('.btn-submit.red').addClass('d-none');
+        setTimeout(() => {
+            $('.btn-submit.grey').addClass('d-none');
+            $('.btn-submit.red').removeClass('d-none');
+        }, 300);
+    })
+    $('.btn-change-quantity').on('click', function(event) {
```
Comment (tamnguyentps) [inline app/Block/cart.twig:23]: Missing "off('click')"

File: app/Block/cart.twig [hunk 3]
```diff
@@ -8,58 +8,77 @@ http://www.lockon.co.jp/
...
+                    $('.btn-submit.red').removeClass('d-none');
+                });
+            }).fail(function(data) {
+                alert('{{ 'front.product.add_cart_error'|trans }}');
+            })
+        ;
+    });
+});
+</script>
+<div class="cart-trash cart">Thông tin đơn hàng </div>
```
Comment (tamnguyentps) [inline app/Block/cart.twig]: Class cart-trash but label is "Thông tin đơn hàng"

**Review by tamnguyentps — CHANGES_REQUESTED** [review]
@duylong192 I have some feedbacks for you

**Review by appdoctor92 — APPROVED** [review]

**Pull request merged**